Makefile

```diff
@@ -16,7 +16,13 @@ EXECUTABLE_FILES=$(shell find . -type f -executable | egrep -v '^\./(website/[ve
 GIT_DIRTY=$(shell test -n "`git status --porcelain`" && echo "+CHANGES" || true)
 GIT_COMMIT=$(shell git rev-parse --short HEAD)
 GIT_IMPORT=github.com/hashicorp/packer/version
-GOLDFLAGS=-X $(GIT_IMPORT).GitCommit=$(GIT_COMMIT)$(GIT_DIRTY)
+UNAME_S := $(shell uname -s)
+LDFLAGS = ""
+ifeq ($(UNAME_S),Linux)
+	LDFLAGS=-linkmode external -extldflags -static
+endif
+
+GOLDFLAGS="$(LDFLAGS) -X $(GIT_IMPORT).GitCommit=$(GIT_COMMIT)$(GIT_DIRTY)"
```

Comment:
While testing I noticed that the git commit information is no longer injected into the version string

Binaries built on master
```
⇶  ./pkg/linux_amd64/packer version
Packer v1.5.6-dev (c35868241+CHANGES)
```
Binaries build of branch
```
⇶  ./pkg/linux_amd64/packer version
Packer v1.5.6-dev
```

Reply:
Looks like the quotes around the flags were causing issues for gox